docs/pre-commit.md

```diff
@@ -0,0 +1,14 @@
+# Using pre-commit locally
+
+### 3 versions of every file that make up our git authoring lifecycle (HEAD, Staged, Unstaged)
+
+### Running the pre-commit hook directly from the .git folder
+- run `bash .git/hooks/pre-commit` to execute pre-commit hook without committing
```

Comment:
I think I figured out why this didn't work as expected for me yesterday--I believe this command runs the hook only on the files that are staged, and I didn't have anything staged at the time. If you want to run pre-commit on everything in the repo regardless of staging, it's `re-commit run --all-files`

Reply:
Exactly! Let's get that added to the docs so it's clear when we'd choose to run each command?